unabridged-works-volume-2/original/01-root-of-popery-struck.adoc

```diff
@@ -176,24 +183,29 @@ without the Spirit`'s infallible opening of his own words to him.
 So that the Spirit itself is the ground and foundation of
 all true light and knowledge of the things of God.
 
-PROP.
-IV. That if there be any light to be found any where shining from God,
+[.old-style]
+=== Prop. IV.
```

Comment:
Okay so this a series of propositions that all seem to have a "preface" except this one. Idk if I did them right making the first sentence kind of into a description of the proposition, or if I should have just left them as plain text. Or just leave it how it is with the 4th one not having one. idk!!